docs/linux_dependencies.md

```diff
@@ -127,8 +127,10 @@ mkShell rec {
     libxkbcommon wayland # To use the wayland feature
   ];
   LD_LIBRARY_PATH = lib.makeLibraryPath buildInputs;
+  VK_ICD_FILENAMES="/run/opengl-driver/share/vulkan/icd.d/radeon_icd.x86_64.json:/run/opengl-driver-32/share/vulkan/icd.d/radeon_icd.i686.json"; # Read below
```

Comment:
that line is probably only valid for some gpu (amd) / cpu (x86), I don't think it should be a general recommendation